### DISCUSSION TITLE
Some questions about UI, about Instant vector and Range vector

### DISCUSSION THREAD

**Discussion by evanzhang87** [grafana/pyroscope#1948] [Q&A] [answered]
Hello,I'm trying to use `pyroscope` to analyze my application, but the UI make me confused.
![image](https://github.com/grafana/pyroscope/assets/60881968/5a300395-5c0d-48d0-9dd4-80127828160a)
As you know, `Total amount of RAM` bar means value of timestamp, it's a `Range vector`, so what does `Table and Flamegraph` mean？
![image](https://github.com/grafana/pyroscope/assets/60881968/ae30a574-280c-4b65-b17c-b5aaf6de3513)
Does it means `sum_over_time()` or `avg_over_time()` ? (or `max/min_over_time()`?  Although I don't think so) .Or just a `Instant vector` of some timestamp?

**Comment by cyriltovena** [comment, marked as answer]
This is a great question and we're tackling this issue starting here https://github.com/grafana/pyroscope/pull/2758

Right now this is a `sum_over_time`, so it's possible each point have multiple collected samples for the same applications. As you noticed in the case of a `in_use` memory this is confusing, we're thinking about moving this to an avg_over_time or max_over_time.

Right now it's a range vector that is selected, we selected multiple point per step, not a single one like Prometheus does with the lookback period. This is also something that we are thinking about.
